packages/wallet-backend-update/src/utils/helpers.ts

```diff
@@ -0,0 +1,20 @@
+export const formatBalance = (value: bigint | number, scale = 2): bigint => {
+  return BigInt(transformAmount(value, scale).toFixed(2))
+}
+
+export const transformAmount = (value: bigint | number, scale = 2) => {
+  return Number(BigInt(value) / BigInt(10 ** scale))
+}
```

Comment:
I don't think `scale` should have a default value

Reply:
@dragosp1011